tests/e2e/common/renew-token.wdio-spec.js

```diff
@@ -0,0 +1,28 @@
+
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const commonPage = require('../../page-objects/common/common.wdio.page');
+const moment = require('moment');
+const chai = require('chai');
+
+describe('should renew token', async () => {
+
+  beforeEach(async () => {
+    await loginPage.cookieLogin();
+    await commonPage.goToPeople();
+  });
+
+  it('Refresh page multiple times and verify token gets renewed', async () => {
+    Array.from(Array(3)).forEach(async () => {
```

Comment:
The array `forEach` doesn't execute async. Meaning it won't wait for any promises to complete even if you have the async tag.  You might also put in an expect message that indicates what count is set to for future troubleshooting. 

`for(let count = 0; count < 3; count++)`

Reply:
done